### PR TITLE
Added  Mouse and Hedgehog

### PR DESCRIPTION
**Pull request by lyaleko**
Please look at this code

### CODE REVIEW

File: src/main/java/tasks/homework/threadtask/Hedgehog.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        System.out.printf(String.format("I am %s! ", this.name));
+        try {
+            Thread.sleep(50);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
```
Comment (vpetrushka) [inline src/main/java/tasks/homework/threadtask/Hedgehog.java:29]: why main is here?

File: src/main/java/tasks/homework/threadtask/Mouse.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        System.out.printf(String.format("%s PEEP ", this.name));
+        try {
+            Thread.sleep(200);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+
```
Comment (vpetrushka) [inline src/main/java/tasks/homework/threadtask/Mouse.java:32]: why main is here?

Comment (lyaleko) [reply src/main/java/tasks/homework/threadtask/Mouse.java]: Main is removed

File: src/main/java/tasks/homework/threadtask/Hedgehog.java (hunk 1 above)
Comment (lyaleko) [reply src/main/java/tasks/homework/threadtask/Hedgehog.java]: Main is removed

**Review by vpetrushka — APPROVED** [review]

**Pull request merged**